===================================================================
RCS file: /pack/anoncvs/sylpheedclaws/plugins/etpan-privacy/src/plugin.c,v
retrieving revision 1.1.1.1.2.10
retrieving revision 1.1.1.1.2.11
diff -u -r1.1.1.1.2.10 -r1.1.1.1.2.11
--- src/plugin.c	2006/12/15 19:06:14	1.1.1.1.2.10
+++ src/plugin.c	2007/06/21 17:12:30	1.1.1.1.2.11
@@ -46,7 +46,7 @@
 {
 	int r;
 
-	if( !check_plugin_version(MAKE_NUMERIC_VERSION(2, 6, 1, 41),
+	if( !check_plugin_version(MAKE_NUMERIC_VERSION(2,9,2,72),
 				VERSION_NUMERIC, _("etPan! Privacy"), error) )
 		return -1;
 	
@@ -63,11 +63,12 @@
 	return 0;
 }
 
-void plugin_done(void)
+gboolean plugin_done(void)
 {
 	etpan_privacy_prefs_done();
 	etpan_privacy_done();
 	debug_print(_("etPan! - Privacy plugin unloaded\n"));
+	return TRUE;
 }
 
 const gchar *plugin_name(void)
